agent-service/tests/roster_import/test_row_parsers_roster.py

```
@@ -0,0 +1,166 @@
+        response = handler.parse_roster_excel(str(temp_excel_path))
+
+        result, issues = response.result, response.issues
+        warnings = [issue for issue in issues if issue.severity == ParseIssueSeverity.WARNING.value]
+
+        assert len(result.entries) == 1
+        assert len(warnings) == 0
+
+class TestTimeRangeDetection:
+    """Tests for time range detection in _parse_time()."""
+
+    def test_hyphen_range_detected(self, handler):
```

Ruff (0.14.14)

[warning] Unused method argument: `handler`

(ARG002)